src/inputs/ObjectSearch.tsx

```diff
@@ -161,7 +162,7 @@ class ObjectSearch extends Component<IObjectSearchProps> {
 
     return (
       <Antd.Select.Option className={className} key={ITEM_KEYS.ADD}>
-        <div>{addNewContent || <><Antd.Icon type='plus' /> Add new</>}</div>
+        <div>{addNewContent || <>Can't find it? <a>Add new {label}</a></>}</div>
```

Comment:
So before it'd just be a place to add a new object search create field without a label?

Reply:
Before it did the same thing: It's the button to say none of these search results are what I want, so I want to create a new one.

The text is just slightly different. Kevin in Registry kept overriding this text instead of just fixing it in the library. This will allow me to delete those overrides.